app/actions/filterActions.js

```diff
@@ -0,0 +1,8 @@
+
+import { createAction } from 'redux-actions';
+
+export const filterByName = createAction('FILTER_BY_NAME');
+export const filterByType = createAction('FILTER_BY_TYPE');
+export const filterByPremises = createAction('FILTER_BY_PREMISES');
+export const filterByPeople = createAction('FILTER_BY_PEOPLE');
+export const filterByOpenNow = createAction('FILTER_BY_OPEN_NOW');
```

Comment:
There shouldn't be a need for 5 different actions to sort search results. One sort action should be enough. Also, why are these actions in a totally different place than the rest of the search redux code? I would assume that the sorting should be attached to the `searchResources` action that already exists.